sdk/keyvault/Azure.Security.KeyVault.Administration/tests/BackupRestoreTestBase.cs

```diff
@@ -17,6 +17,7 @@ public abstract class BackupRestoreTestBase : AdministrationTestBase
 
         internal string SasToken { get; private set; }
         internal string BlobContainerName = "backup";
+        internal string BlobContainerNameMultiPart = "backup/some/folder/name";
```

Comment:
Rather than re-record a bunch of large JSON files that will expand our git database further, could you just add some unit tests with a bunch of variants (with and without trailing slashes, etc.)? Much faster, and much less impact to the git database.

Reply:
I assume we’d still want to have a live test at least? The parsing logic is good to unit test, but the test that the service handles paths like this properly is more important.

Reply:
Decided to just add `[LiveOnly]` and delete the two extra recordings.